src/app/core/services/http-rundown.service.ts

```diff
@@ -15,27 +14,16 @@ export class HttpRundownService implements RundownService {
   constructor(
     private readonly http: HttpClient,
     private readonly httpErrorService: HttpErrorService,
-    private readonly entityParser: EntityParser,
-    private readonly responseParser: ResponseParser
+    private readonly entityParser: EntityParser
   ) {}
 
   public fetchRundown(rundownId: string): Observable<Rundown> {
-    return this.http.get<unknown>(`${RUNDOWN_URL}/${rundownId}`).pipe(
-      map(this.responseParser.parseResponse.bind(this.responseParser)),
-      map(this.assertType.bind(this)),
+    return this.http.get<{ data: unknown }>(`${RUNDOWN_URL}/${rundownId}`).pipe(
```

Comment:
It would be nice not needing to write `{ data: unknown }` for all our requests.
Since every response now has a `data` attribute we could make an interface like so:
```
interface HttpResponse<T> {
  data: T
}
```
and use it like so:
```
return this.http.get<HttpResponse<Rundown>>(SOME_URL)
```
This also has the benefit of already mapping the result to the desired type.

For a place to place this interface an idea could be to have a `HttpBaseService` that all our `HttpXServices` extends from.

Reply:
Suggested interface has been added.
Have not made the base class, as it would add little to no reusable code between classes.